source/_integrations/teslemetry.markdown

```diff
@@ -194,3 +194,73 @@ These are the entities available in the Teslemetry integration. Not all entities
 ## Vehicle sleep
 
 Constant API polling will prevent most Model S and Model X vehicles manufactured before 2021 from sleeping, so the Teslemetry integration will stop polling these vehicles for 15 minutes, after 15 minutes of inactivity. You can call the `homeassistant.update_entity` service to force polling the API, which will reset the timer.
+
+## Services
+
+Teslemetry provides various custom services to interact with the Tesla Fleet API directly.
+
+### Navigate to coordinates
+
+`teslemetry.navigation_gps_request`
+
+| Field         | Description                | Example                          |
+|---------------|----------------------------|----------------------------------|
+| device_id     | The vehicle's device_id    | 0d462c0c4c0b064b1a91cdbd1ffcbd31 |
+| gps           | Dictionary of coordinates  |                                  |
+| gps.latitude  | Latitude in degrees        | -27.9699373                      |
+| gps.longitude | Longitude in degrees       | 153.4081865                      |
+| order         | Order for this destination | 1                                |
```

Comment:
What does order mean in this context? If the number is 3, for example, would this be the third stop on your route to destination X?

Reply:
I think so... Tesla isn't very clear about this themselves.

Reply:
Oh, I see. Probably, better leave the original terminology then, like you did.